test/unit/yesno.spec.ts

```diff
@@ -220,6 +220,41 @@ describe('Yesno', () => {
       expect(yesno.intercepted()).to.have.lengthOf(1);
     });
 
+    it('should support mock override with respond', async () => {
```

Comment:
ThIs looks good. Can you just move it to the test section for `respond` then go ahead and merge?